### PR TITLE
[WIP] CHE-1391: rework list of workspaces

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova  WDYT?
selection_047

Signed-off-by: Oleksii Orel oorel@codenvy.com

### CODE REVIEW

File: dashboard/src/app/colors/che-colors.styl [hunk 1]
```diff
@@ -24,6 +24,13 @@ $frame-subscription-bg-color = alpha($che-medium-blue-color, 0.2)
 $auth-error-color = $che-orange-color
 $auth-ok-color = $che-green-color
 
+$list-blue-color = $che-medium-blue-color
```
Comment (ashumilova) [inline dashboard/src/app/colors/che-colors.styl]: $list-blue-color - let the property describe part of list it is used for (border, selection, background, etc)

File: dashboard/src/app/index.styl [hunk 2]
```diff
@@ -232,3 +232,7 @@ md-option:hover
 
 .logo-color-white .outer-fill
   fill $che-white-color
+
+.noselect
+  -webkit-user-select none
```
Comment (ashumilova) [inline dashboard/src/app/index.styl]: you can provide just : 
`user-select none`

File: dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.controller.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * @ngdoc controller
+ * @name workspaces.list.controller:WorkspaceItemStatusController
+ * @description This class is handling the controller for listing the workspaces
```
Comment (ashumilova) [inline dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.controller.js]: for listing the workspaces - wrong description

File: dashboard/src/app/colors/che-colors.styl (hunk 1 above)
Comment (olexii4) [inline dashboard/src/app/colors/che-colors.styl]: OK
We have the same color for list header elements(background for buttons, border and placeholder for search widget and header text). It should be the same color for all of those elements.  I have change it to $list-header-blue-color

File: dashboard/src/app/index.styl (hunk 2 above)
Comment (olexii4) [inline dashboard/src/app/index.styl]: OK

File: dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.controller.js (hunk 3 above)
Comment (olexii4) [inline dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.controller.js]: OK

**Comment by slemeur** [comment]
After reviewing the work done on the new list, the overall work is nice. There are some little details that are needing some more work (ordered by priority):
- [x] On the dashboard: remove the "Recent Projects" widget
- [x] On the list of workspaces: Use the same typo between "Add Workspace" and "SEARCH". Let's make the two buttons with first capitalized letters
- [x] On the list of workspaces: the button “Add Workspace” does not seem to be vertically centered
- [ ] On workspace details: the list of workspace members in "Share" has not been revisited
- [x] On workspace details: the position of the list’s header buttons is not properly aligned (as we did in workspaces lists)
- [x] On the list of workspaces: the tooltip displayed on hovering the workspace name should show the relative time when the workspace has been used. Instead of "Last modified: Friday 12th August", it should be "Last used: 2 days ago"

**Comment by slemeur** [comment]
Adding feedbacks:
- [ ] On the list of workspace, when hovering a running workspace say "Running since xxx ."

**Comment by olexii4** [comment]
"Running since" - It isn't implemented at server side,  but we can add it as a new attribute. WDYT?

File: dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.styl [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  color inherit
+  outline none
+  height inherit
+
+  span
+   outline none
+
+.workspaces-busy
+  opacity 0.4
+  $disabled-color
```
Comment (akurinnoy) [inline dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.styl]: CSS property is missed

**Comment by slemeur** [comment]
@olexii4 : would be nice to get the value for the "running since". but that is not needed for that PR.

File: dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.styl (hunk 4 above)
Comment (olexii4) [inline dashboard/src/app/workspaces/list-workspaces/workspace-status-action/workspace-status.styl]: OK

**Pull request merged**